packages/fast-components-styles-msft/src/progress/index.ts

```diff
@@ -0,0 +1,105 @@
+import designSystemDefaults, { IDesignSystem, withDesignSystemDefaults } from "../design-system";
+import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
+import { IMSFTProgressClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { ensureBrandNormal, largeContrast } from "../utilities/colors";
+import Chroma from "chroma-js";
+import { get } from "lodash";
+
+/* tslint:disable-next-line */
+const styles: ComponentStyles<IMSFTProgressClassNameContract, IDesignSystem> = (config: IDesignSystem): ComponentStyleSheet<IMSFTProgressClassNameContract, IDesignSystem> => {
+    const designSystem: IDesignSystem = withDesignSystemDefaults(config);
+    const brandColor: string = ensureBrandNormal(config);
+    const determinateBackgroundColor: string = largeContrast(designSystem.contrast, designSystem.foregroundColor, designSystem.backgroundColor);
+
+    return {
+        progress: {
+            display: "flex",
+            width: "100%",
+            alignItems: "center",
+            height: "10px",
+            textAlign: "left"
+        },
+        progress_determinateValueIndicator: {
+            background: brandColor,

```

Comment:
@nicholasrice do you think we should use the contrast ratios here to ensure the brand color is meeting contrast ration with the determinate background color?

Reply:
I don't think so. If you look at the component, the accent color (value indicator) is actually on top of the page background. It appears to be filling the bar created by the grey, however it is only on the right edge that the indicator is against the grey - all 3 other edges are on the page background.